lib/nerves_network/dhcp_manager.ex

```diff
@@ -20,16 +21,21 @@ defmodule Nerves.Network.DHCPManager do
   end
 
   def init({ifname, settings}) do
-    # Register for nerves_network_interface and dhclient events
-    {:ok, _} = Registry.register(Nerves.NetworkInterface, ifname, [])
-    {:ok, _} = Registry.register(Nerves.Dhclientv4, ifname, [])
+    # When the ifname network interface had already been registered with registries Nerves.NEtworkInterface or
+    # Nerves.Dhclient that would not be a problem hence we can merely ignore the :already_registered :error
+    # results.
+    rreg_res1 = Registry.register(Nerves.NetworkInterface, ifname, []) |> Utils.Registry.digest_register_results()
+    rreg_res2 = Registry.register(Nerves.Dhclientv4, ifname, []) |> Utils.Registry.digest_register_results()
+
+    Logger.debug ".init   rreg_res1 = #{inspect rreg_res1}"
+    Logger.debug ".init   rreg_res2 = #{inspect rreg_res2}"
```

Comment:
I think we have the logger already setup to print the function name, so you might want to ommit the `".init"`

Reply:
yup - got a rid of those